lib/ain-grpc/src/rpc/eth.rs

```diff
@@ -1125,14 +1129,10 @@ impl MetachainRPCServer for MetachainRPCModule {
         let logs = self
             .handler
             .filters
-            .get_filter_logs_from_id(filter_id, curr_block)
+            .get_logs_filter_from_id(filter_id, curr_block)
```

Comment:
Doesn't the new name imply that we are getting the filter, but we are actually getting the logs of the filter here? Perhaps `get_logs_from_filter_id` would be best.